indent/javascript.vim

```diff
@@ -448,7 +448,22 @@ function GetJavascriptIndent()
     else
       call cursor(v:lnum, vcol)
     end
-  endif
+elseif line =~ ')' || line =~ s:comma_last && line !~ s:var_stmt
+    let counts = s:LineHasOpeningBrackets(lnum)
+    if counts[0] == '2'
+      " let curpos = getpos(".")
+      call cursor(lnum, 1)
+      " Search for the opening tag
+      let mnum = searchpair('(', '', ')', 'bW', s:skip_expr)
+      "Restore the cursor position
+      " call cursor(curpos)
```

Comment:
This bottom section is relevant to fix a few bugs #315 and #57 and #202 though when I coded most of it I didn't see that improperly escaped regex near the top which changed some behavior so possibly some is redundant now